mne/viz/backends/renderer.py

```diff
@@ -4,29 +4,31 @@
 #          Eric Larson <larson.eric.d@gmail.com>
 #          Joan Massich <mailsik@gmail.com>
 #          Guillaume Favelier <guillaume.favelier@gmail.com>
+#          Oleh Kozynets <ok7mailbox@gmail.com>
 #
 # License: Simplified BSD
 
 import importlib
 from contextlib import contextmanager
 import sys
 
-from ._utils import _get_backend_based_on_env_and_defaults, VALID_3D_BACKENDS
+from ._utils import Backends3D
 from ...utils import logger
-from ...utils.check import _check_option
 
 try:
     MNE_3D_BACKEND
     MNE_3D_BACKEND_TEST_DATA
 except NameError:
-    MNE_3D_BACKEND = _get_backend_based_on_env_and_defaults()
+    MNE_3D_BACKEND = Backends3D.get_backend_based_on_env_and_defaults()
```

Comment:
More in `Backends3D`, if you are using this just for having context in your testing (which I love it) you could just use the `BunchConst` directly: 

```py
def foo(): 
     print('hi') 

backends3D = mne.utils.BunchConst(mayavi='mayavi', foo=foo)
backends3D.foo()
```